cogs/gatekeep/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "author": ["owari_da"],
+    "name": "Gatekeep",
+    "short": "Automatically ban scammers that send messages with user-defined keywords",
+    "description": "Automatically detects and bans scammers that send messages containing user-defined keywords.",
+    "end_user_data_statement": "This cog will store users' Discord ID of those who joined the server less than 14 days ago.",
```

Comment:
Can you remove the 14 day part since this is user configurable? I think leaving it generic with "This cog will store users' Discord IDs" is sufficient.